kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -145,30 +144,25 @@
         // creates array of words in filter, removes empty strings
         const searchFilter = this.searchFilter.split(' ').filter(Boolean).map(
           // returns an array of search parameters, ignoring case
-          (query) => new RegExp(query, 'i'));
+          (query) => {
+            const escapedQuery = query.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&');
+            return new RegExp(escapedQuery, 'i');
+          });
 
         return this.users.filter((user) => {
           // fullname created using es6 templates
           const names = [user.full_name, user.username];
 
-          let hasRole = true;
+          let isKind = true;
           let hasName = true;
```

Comment:
also for line 156, I think I was thinking of these in the context of this loop iteration; we're looking at a user, this user `is` of `Kind` x, and `has` the `Name` y. 

But I see what you're saying. I'll change it.